Classes/Service/EmailService.php

```diff
@@ -54,4 +63,38 @@ public function informEmployeeAboutTheirData(array $employee, string $content):
             $mail->send();
         }
     }
+
+    public function sendEditMail(Employee $employee, $request): void
+    {
+        $view = GeneralUtility::makeInstance(StandaloneView::class);
```

Comment:
OK for now, but for more used extensions (community) I would prefer using FluidEmail